generate_build_config.py

```diff
@@ -51,6 +51,61 @@
 - chroot $CHROOT_ROOT apt-get -y update
 """  # noqa: E501
 
+SETUP_CONTENT = b"""\
+#!/bin/sh -eux
+mv /usr/sbin/grub-probe /usr/sbin/grub-probe.dist
+cat <<"PSUEDO_GRUB_PROBE" > /usr/sbin/grub-probe
+#!/bin/sh
+Usage() {
+   cat <<EOF
+Usage: euca-psuedo-grub-probe
```

Comment:
It's "pseudo", the original author had a typo. Also that's not the name here.  You could just make this a comment rather than a usage function or you could document the options to make this meaningful.